integration/helpers/base_test.py

```diff
@@ -351,3 +370,32 @@ def verify_stack(self):
         error = verify_stack_resources(self.expected_resource_path, self.stack_resources)
         if error:
             self.fail(error)
+
+    def verify_response(self, url, expected_status_code):
```

Comment:
We should maybe name this: `verify_get_request_response`.
You should also add docstrings

Reply:
make sense to me, will update it